src/client/components/LogView.tsx

```diff
@@ -59,6 +66,48 @@ class LogView extends Component<LogViewProps, LogViewState> {
     })
   }
 
+  renderCollapseButton = (): JSX.Element => {
```

Comment:
Instead of this function you could just add this component directly to render:
```
<CollapseButton
    label="Expand"
    onClick={() => this.handleCollapseChange(!this.state.collapsed)}
/>
```